src/Main.elm

```diff
@@ -25,42 +90,118 @@ init =
 
 type Msg
     = NoOp
+    | TabsMsg Tabs.Msg
+    | ContactDetailsMsg ContactDetails.Msg
+    | SettingsMsg Settings.Msg
+    | TagsMsg Tags.Msg
 
 
 update : Msg -> Model -> ( Model, Cmd Msg )
-update _ model =
-    ( model, Cmd.none )
+update msg model =
+    case model of
+        Success ({ section, userGroup } as successData) ->
+            case ( msg, section ) of
+                ( TabsMsg tabsMsg, _ ) ->
+                    let
+                        ( updatedTabsModel, cmd ) =
+                            Tabs.update tabsMsg
+                    in
+                    ( Success { successData | section = initSection updatedTabsModel }
+                    , Cmd.map TabsMsg cmd
+                    )
```

Comment:
I wonder why is this part here needed if you are dealing with `TabsMsg` later in the case anyways.